core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -782,6 +785,22 @@ oppia.directive('conversationSkin', ['urlService', function(urlService) {
         $scope.collectionId = GLOBALS.collectionId;
         $scope.collectionTitle = GLOBALS.collectionTitle;
 
+        if ($scope.collectionId) {
+          $scope.collectionDataUrl = UrlInterpolationService.interpolateUrl(
+            COLLECTION_DATA_URL_TEMPLATE, {
+              collection_id: $scope.collectionId
+            });
+          $http.get($scope.collectionDataUrl).then(
+            function(response) {
+              $scope.collectionSummary = response.data.collection_summary[0];
```

Comment:
The function `summary_services.get_displayable_collection_summary_dicts_matching_ids(collection_id)` returns a list of collection summaries